packages/teleterm/src/ui/appContext.ts

```diff
@@ -147,4 +149,25 @@ export default class AppContext implements IAppContext {
       );
     });
   }
+
+  private showConfigParsingErrors(): void {
+    const errors = this.mainProcessClient.configService.getStoredConfigErrors();
+    if (errors) {
+      errors.forEach(error => {
+        this.notificationsService.notifyError({
+          title: `Invalid config value for key ${error.path[0]}`,
+          description: error.message,
+        });
+      });
```

Comment:
Somehow I managed to not submit a comment about that, even though I vividly remember writing it two days ago, sorry.

---

It's not really that important at the moment as we have only a single config option. But once we add more than a few options, it'd be good to send a single notification for all errors rather than a notification for each error.

Our notification system doesn't handle displaying many notifications that well.

For now I'm fine with leaving a TODO comment there and coming back to it later.

Reply:
Added single notification!